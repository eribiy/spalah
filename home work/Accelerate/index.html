<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Accelerate Long Island</title>
    <link href="styles/reset.css" rel="stylesheet">
    <link href="styles/style.css" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet" type="text/css">
</head>
<body>
    <div class="wrapper">
        <div class="header_wrap">
            <header class="header clearfix">
                <div class="logo_wrap">
                    <a href=""><img src="img/logo.png" class="img_logo"></a>
                </div>
                <nav class="wrapper_nav">
                    <a href="" class="nav">home</a>
                    <a href="" class="nav">about</a>
                    <a href="" class="nav">blog</a>
                    <a href="" class="nav">portfolio</a>
                    <a href="" class="nav">apply</a>
                    <a href="" class="nav">contact us</a>
                </nav>
                <form class="icons_wrap">
                    <input type="submit" class="a_icon" value="" />
                    <input type="submit" class="a_icon" value="" />
                    <input type="submit" class="a_icon" value="" />
                    <input type="submit" class="a_icon" value="" />
                </form>
            </header>
        </div>
        <div class="wrapper_img_1">
            <div class="block_left"></div>
            <div class="block_right"></div>
            <img src="img/1.png" class="img_1">
            <div class="wrapper_text_img_1">
                <h1 class="h_img_1">welcome</h1>
                <p class="text_img_1">accelerate long island is a unique collaboration among long island's world class research institutions
                        and its business community to commercialize research and create an entrepreneurial ecosystem</p>
            </div>
        </div>
        <div class="text_1_bg">
            <div class="wrapper_text_1 clearfix">
                <h1 class="h_text_1">about us</h1>
                <h6 class="h_small_text_1">To get your startup in front of the right people on Long Island</h6>
                <div class="wrapper_column_text_1">
                    <div class="column_text_1">
                        <h2 class="h_column_text_1">accelerate assis</h2>
                        <div class="border"></div>
                        <span class="text_column_text_1">Connecting entrepreneurs with mentors to accelerate the formation of startup companies</span>
                    </div>
                    <div class="column_text_1">
                        <h2 class="h_column_text_1">accelerate ecosystem</h2>
                        <div class="border"></div>
                        <span class="text_column_text_1">Building an entrepreneurial ecosystem</span>
                    </div>
                    <div class="column_text_1">
                        <h2 class="h_column_text_1">accelerate fuding</h2>
                        <div class="border"></div>
                        <span class="text_column_text_1">Apply to our seed fund</span>
                    </div>
                </div>
                <a href="" class="button_1">join the accelerate ecosystem</a>
            </div>
        </div>
        <div class="wrapper_img_2">
            <img src="img/2.png" class="img_2">
            <div class="wrapper_text_img_2">
                <form class="form_img_2">
                    <label for="email" class="label_img_2">Subscribe  to our updates</label>
                    <input id="email"  type="email" placeholder="email">
                    <input class="button_2" type="submit" value="subscribe">
                </form>
            </div>
        </div>
    </div>

</body>
</html>