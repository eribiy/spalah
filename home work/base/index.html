<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title></title>
    <link href="styles/reset.css" rel="stylesheet">
    <link href="styles/style.css" rel="stylesheet">
    <body>
        <div class="wrapper_all">
            <div class="header_bg">
                <header class="header clearfix">
                    <div class="content_header">
                        <a href=""><img class="logo" src="images/logo.png"></a>
                        <p class="slogan">
                            A great new free psd theme to showcase your new application.
                        </p>
                        <div class="wrapper_icons">
                            <a href=""><div class="icon_bg"></div></a>
                            <a href=""><div class="icon_bg"></div></a>
                            <a href=""><div class="icon_bg"></div></a>
                        </div>
                    </div>
                </header>
                <div class="content_bg">
                    <div class="content">
                        <div class="column">
                            <div class="icon_big_bg"></div>
                            <h3 class="h3">Editable Theme</h3>
                            <p class="text_column">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspenendisse fringilla fringilla.</p>
                        </div>
                        <div class="column">
                            <div class="icon_big_bg"></div>
                            <h3 class="h3">Flat Design</h3>
                            <p class="text_column">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse fringilla fringilla.</p>
                        </div>
                        <div class="column">
                            <div class="icon_big_bg"></div>
                            <h3 class="h3">Reach Your Audience</h3>
                            <p class="text_column">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspenendisse fringilla fringilla.</p>
                        </div>
                    </div>
                    <div class="video_bg">
                        <div class="wrapper_video">
                            <div class="content_video">
                                <h3 class="h3_video">Get Notified Of Any Updates!</h3>
                                <p class="text_column text_video">
                                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse fringilla fringilla nisl congue congue. Maecenas nec condimentum libero, at elementum mauris. Phasellus eget nisi dapibus, ultricies nisl at, hendrerit risusuis ornare luctus id sollicitudin ante lobortis at.
                                </p>
                                <form class="form_email">
                                    <input type="email" class="email" placeholder="email address" />
                                    <input type="submit" class="button" value="notify"/>
                                </form>
                            </div>
                            <div class="wrap_video">
                                <iframe class="video" src="https://www.youtube.com/embed/TD4h9Fdn3-o" frameborder="0" allowfullscreen></iframe>
                            </div>
                        </div>
                        <div class="content_2_bg">
                            <div class="content_2">

                            </div>

                        </div>

                    </div>
                </div>
            </div>

        </div>

        <script src="js/scripts.js"></script>
    </body>
</html>