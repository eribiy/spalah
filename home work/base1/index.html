<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title></title>
    <link rel="stylesheet" type="text/css" href="slick/slick.css"/>
    <link rel="stylesheet" type="text/css" href="slick/slick-theme.css"/>
    <link href="styles/reset.css" rel="stylesheet">
    <link href="styles/style.css" rel="stylesheet">
    <body>
    <div class="center">
        <div class="wrap_img"><img class="img_slider" id="img1" src="images/photo1.jpg"></div>
        <div class="wrap_img"><img class="img_slider" id="img2" src="images/photo2.jpg"></div>
        <div class="wrap_img"><img class="img_slider" id="img3" src="images/photo3.jpg"></div>
        <div class="wrap_img"><img class="img_slider" src="images/photo4.jpg"></div>
        <div class="wrap_img"><img class="img_slider" src="images/photo10.jpg"></div>
        <div class="wrap_img"><img class="img_slider" src="images/photo6.jpg"></div>
        <div class="wrap_img"><img class="img_slider" src="images/photo7.jpg"></div>
        <div class="wrap_img"><img class="img_slider" src="images/photo8.jpg"></div>
        <div class="wrap_img"><img class="img_slider" src="images/photo3.jpg"></div>
        <div class="wrap_img"><img class="img_slider" src="images/photo10.jpg"></div>
    </div>
    <script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.min.js"></script>
    <script type="text/javascript" src="//code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
    <script type="text/javascript" src="slick/slick.min.js"></script>
    <script src="js/scripts.js"></script>
    </body>
</html>