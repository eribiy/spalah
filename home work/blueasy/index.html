<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title></title>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
    <link href="styles/reset.css" rel="stylesheet">
    <link href="styles/style.css" rel="stylesheet">
<body>
<div class="main-container">
    <section class="main-bg">
        <div class="header-bg">
            <header class="header clearfix">
                <a href="/" class="logo">blu<span>e</span>asy</a>
                <nav class="nav">
                    <a href="#" class="nav-item active">home</a>
                    <a href="#" class="nav-item">services</a>
                    <a href="#" class="nav-item">features</a>
                    <a href="#" class="nav-item">portfolio</a>
                    <a href="#" class="nav-item">contact</a>
                </nav>
                <div class="mobile-nav-wrapper clearfix">
                    <a role="button" class="mobile-btn">
                        <span></span>
                    </a>
                    <nav class="mobile-nav">
                        <a href="#" class="mobile-nav-item active">home</a>
                        <a href="#" class="mobile-nav-item">services</a>
                        <a href="#" class="mobile-nav-item">features</a>
                        <a href="#" class="mobile-nav-item">portfolio</a>
                        <a href="#" class="mobile-nav-item">contact</a>
                    </nav>
                </div>
            </header>
        </div>
        <h2 class="main-title m-main-title_width">“I’m looking for the unexpected.</h2>
        <h2 class="main-title">I’m looking for things I’ve never seen before.” </h2>
    </section>
    <section class="services">
        <div class="section-inner">
            <h3 class="section-title">Services</h3>
            <div class="section-row">
                <div class="section-col">
                    <img class="services-img" src="images/coffee.png">
                    <h4 class="services-title">Coffee</h4>
                    <p class="services-content">Apple pie icing sweet. Brownie jelly-o applicake applicake sweet roll
                        liquorice bear claw. Jujubes carrot cake cotton candy sweet tart brownie. Tiramisu applicake
                        jujubes.</p>
                </div>
                <div class="section-col">
                    <img class="services-img" src="images/instant.png">
                    <h4 class="services-title">Instant</h4>
                    <p class="services-content">Apple pie icing sweet. Brownie jelly-o applicake applicake sweet roll
                        liquorice bear claw. Jujubes carrot cake cotton candy sweet tart brownie. Tiramisu applicake
                        jujubes.</p>
                </div>
                <div class="section-col">
                    <img class="services-img" src="images/serious.png">
                    <h4 class="services-title">Serious</h4>
                    <p class="services-content">Apple pie icing sweet. Brownie jelly-o applicake applicake sweet roll
                        liquorice bear claw. Jujubes carrot cake cotton candy sweet tart brownie. Tiramisu applicake
                        jujubes.</p>
                </div>
                <div class="section-col">
                    <img class="services-img" src="images/frame.png">
                    <h4 class="services-title">Frame</h4>
                    <p class="services-content">Apple pie icing sweet. Brownie jelly-o applicake applicake sweet roll
                        liquorice bear claw. Jujubes carrot cake cotton candy sweet tart brownie. Tiramisu applicake
                        jujubes.</p>
                </div>
            </div>
        </div>

    </section>
    <section class="portfolio">
        <div class="section-inner">
            <h3 class="section-title">Portfolio</h3>
            <div class="section-row">
                <div class="section-col">
                    <div class="portfolio-item">
                        <img src="images/img1.jpg" alt="#"/>
                        <span>+</span>
                    </div>
                </div>
                <div class="section-col">
                    <div class="portfolio-item">
                        <img src="images/img2.jpg" alt="#"/>
                        <span>+</span>
                    </div>
                </div>
                <div class="section-col">
                    <div class="portfolio-item">
                        <img src="images/img3.jpg" alt="#"/>
                        <span>+</span>
                    </div>
                </div>
                <div class="section-col">
                    <div class="portfolio-item">
                        <img src="images/img4.jpg" alt="#"/>
                        <span>+</span>
                    </div>
                </div>
                <div class="section-col">
                    <div class="portfolio-item">
                        <img src="images/img5.jpg" alt="#"/>
                        <span>+</span>
                    </div>
                </div>
                <div class="section-col">
                    <div class="portfolio-item">
                        <img src="images/img6.jpg" alt="#"/>
                        <span>+</span>
                    </div>
                </div>
                <div class="section-col">
                    <div class="portfolio-item">
                        <img src="images/img7.jpg" alt="#"/>
                        <span>+</span>
                    </div>
                </div>
                <div class="section-col">
                    <div class="portfolio-item">
                        <img src="images/img8.jpg" alt="#"/>
                        <span>+</span>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section class="video">
        <div class="section-inner">
            <div class="video-row">
                <div class="video-col clearfix">
                    <h3 class="section-title">Just default Section</h3>
                    <p>Bear claw marzipan bear claw applicake I love muffin. Lemon drops gummi bears pastry gummi
                        bears
                        sesame
                        snaps
                        I love unerdwear.com. Soufflé cotton candy dessert candy ice cream wafer gummies cheesecake
                        brownie.
                    </p>
                    <p>Muffin chupa chups jelly beans sweet pie applicake. Croissant chocolate cake I love pudding.
                        Ice
                        cream I
                        love
                        powder pudding apple pie marshmallow. Cupcake marzipan oat cake bonbon I love candy canes
                        toffee.
                    </p>
                    <form class="video-btn-wrapper">
                        <button class="video-btn">visit me</button>
                    </form>
                </div>
                <div class="video-col">
                    <iframe class="video-item" src="https://player.vimeo.com/video/22439234" frameborder="0"
                            webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
                </div>
            </div>
        </div>
    </section>
    <section class="twitter">
        <div class="section-inner">
            <div class="twitter-row">
                <div class="twitter-col">
                    <h3 class="section-title">twitter</h3>
                </div>
                <div class="twitter-col">
                    <i class="fa fa-twitter" aria-hidden="true"></i>
                </div>
            </div>
            <div class="twitter-row">
                <p>Unerdwear cookie liquorice. Cake donut cupcake lollipop soufflé candy. Chocolate oat cake @<span>cheesecake</span>
                    tootsie roll.
                </p>
            </div>
        </div>
    </section>
    <section class="john">
        <div class="section-inner">
            <h3 class="section-title">Who is john doe?</h3>
            <div class="john-row">
                <div class="john-col">
                    <p>Bear claw marzipan bear claw applicake I love muffin. Lemon drops gummi bears pastry gummi bears
                        sesame snaps I love unerdwear.com. Soufflé cotton candy dessert candy ice cream wafer gummies
                        cheesecake brownie.
                    </p>
                    <p>Muffin chupa chups jelly beans sweet pie applicake. Croissant chocolate cake I love pudding. Ice
                        cream I love powder pudding apple pie marshmallow. Cupcake marzipan oat cake bonbon I love candy
                        canes toffee.
                    </p>
                </div>
                <div class="john-col">
                    <p>Pudding dessert jujubes tiramisu gingerbread croissant tiramisu applicake. Sesame snaps sugar
                        plum cotton candy chocolate bonbon lemon drops candy canes cotton candy. Cake toffee pie bear
                        claw pastry.
                    </p>
                    <div class="social-wrapper">
                        <a class="social" href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
                        <a class="social" href="#"><i class="fa fa-google-plus" aria-hidden="true"></i></a>
                        <a class="social" href="#"><i class="fa fa-dribbble" aria-hidden="true"></i></a>
                        <a class="social" href="#"><i class="fa fa-coffee" aria-hidden="true"></i></a>
                    </div>
                </div>
            </div>
        </div>
    </section>
    <section class="test">
        <div class="section-inner">
            <div class="test-row">
                <div class="test-col">
                    <h3 class="section-title">Testimonials</h3>
                </div>
                <div class="test-col">
                    <i class="fa fa-quote-right" aria-hidden="true"></i>
                </div>
            </div>
            <div class="test-text-wrapper">
                <p>Applicake chocolate cake wafer toffee pie soufflé wafer. Tart marshmallow wafer macaroon cheesecake
                    jelly. Gingerbread cookie soufflé sweet roll sweet roll jelly-o.
                </p>
                <span>-Walter White</span>
            </div>
        </div>
    </section>
    <section class="contact-form">
        <div class="section-inner">
            <h3 class="section-title">Contact</h3>
            <form>
                <div class="form-row">
                    <div class="form-col">
                        <label class="form-item">
                            <span>full name</span>
                            <input type="text" class="form-input">
                        </label>
                        <label class="form-item">
                            <span>email</span>
                            <input type="email" class="form-input">
                        </label>
                    </div>
                    <div class="form-col">
                        <p>Jujubes bear claw croissant wafer sweet lemon drops brownie chocolate bar. Sweet lollipop
                            halvah tootsie roll cake fruitcake. Chocolate cake chocolate bar gingerbread lollipop
                            tart.
                            Jujubes bear claw donut chupa chups.</p>
                        <span>Linda Newman</span>
                        <span>W325 State Road 123</span>
                        <span>Mondovi, WI(Wisconsin) 98746-54321</span>
                        <span>(715) 946 - 1234</span>
                    </div>
                </div>
                <textarea class="form-input"></textarea>
                <input type="submit" value="message" class="form-btn"/>
            </form>
        </div>
    </section>
    <div class="footer-bg">
        <footer class="footer">
            <div class="footer-row">
                <div class="footer-col">
                    <ul>
                        <li><a href="#" class="footer-nav-item">home</a></li>
                        <li><a href="#" class="footer-nav-item">services</a></li>
                        <li><a href="#" class="footer-nav-item">portfolio</a></li>
                        <li><a href="#" class="footer-nav-item">contact</a></li>
                    </ul>
                </div>
                <div class="footer-col">
                    <span>2013 &copy</span>
                </div>
            </div>
        </footer>
    </div>
</div>
<script src="js/jquery-1.12.4.min.js"></script>
<script src="js/scripts.js"></script>
</body>
</html>