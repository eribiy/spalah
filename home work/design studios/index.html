<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title>Design Studios</title>
    <link href="styles/reset.css" rel="stylesheet">
    <link href="styles/style.css" rel="stylesheet">
<body>
<div class="wrapper-all">
    <header class="header">
        <div class="header-content">
            <div class="logo-wrapper">
                <a class="logo" href="">design studios</a>
            </div>
            <ul class="menu_wrapper">
                <li class="menu-item"><a href="">home</a></li>
                <li class="menu-item"><a href="">about us</a></li>
                <li class="menu-item"><a href="">services</a></li>
                <li class="menu-item"><a href="">portfolio</a></li>
                <li class="menu-item"><a href="">contact us</a></li>
            </ul>
            <div class="mobile-nav-wrapper clearfix">
                <a role="button" class="mobile-btn">
                    <span></span>
                </a>
                <ul class="mobile-nav">
                    <li class="mobile-nav-item"><a href="">home</a></li>
                    <li class="mobile-nav-item"><a href="">about us</a></li>
                    <li class="mobile-nav-item"><a href="">services</a></li>
                    <li class="mobile-nav-item"><a href="">portfolio</a></li>
                    <li class="mobile-nav-item"><a href="">contact us</a></li>
                </ul>
            </div>
        </div>
    </header>
    <div class="section-wrapper">
        <div class="content-wrapper">
            <section class="description">
                <h1 class="h1-description">clean <span class="h1-description-amp">&amp;</span> Professional web design
                </h1>
                <h2 class="h2-description">providing high quality web solutions</h2>
                <p class="text-description">We provide <span class="text-description-bold">first class</span> web
                    solutions for organizations and small businesses looking to make a <span
                            class="text-description-bold">big impact</span> online.
                    Our services include web design, online marketing campaigns, and search engine optimization.
                </p>
            </section>
            <section class="columns">
                <div class="col">
                    <h3 class="h3-col">web design</h3>
                    <p class="text-col">We provide web design solutions of the very highest order. We’ll work with
                        you from the start to understand your exact requirements and will continuosly work with
                        you to ensure that you’re happy with the final site that you purchase.
                    </p>
                    <a href="" class="button-col">get more details &gt;&gt;</a>
                </div>
                <div class="col">
                    <h3 class="h3-col">marketing</h3>
                    <p class="text-col">We can help you promote your business in both the online and offline worlds.
                        We specialize in a range of fields from viral marketing campaigns to managing paid advertising
                        accounts for you. Your brand will be in safe hands with us.
                    </p>
                    <a href="" class="button-col">view case studies &gt;&gt;</a>
                </div>
                <div class="col">
                    <h3 class="h3-col">optimization</h3>
                    <p class="text-col">Want to drive more traffic to your site? You’ve come the right place! We
                        can increase traffic to your site in an organic and effective manner that will ensure more
                        people are aware of your business and the products/services you offer.
                    </p>
                    <a href="" class="button-col">learn more &gt;&gt;</a>
                </div>
                <div class="col">
                    <h3 class="h3-col">web design</h3>
                    <p class="text-col">We provide web design solutions of the very highest order. We’ll work with
                        you from the start to understand your exact requirements and will continuosly work with
                        you to ensure that you’re happy with the final site that you purchase.
                    </p>
                    <a href="" class="button-col">get more details &gt;&gt;</a>
                </div>
            </section>
            <section class="additional-info">
                <div class="clients-wrapper">
                    <h4 class="h-add">Client Testimonials</h4>
                    <div class="client clearfix">
                        <img class="photo-client" src="img/photo7.jpg">
                        <div class="quote-wrapper">
                            <p class="quote">“Design studios created a beautiful website and were a pleasure to work
                                with during
                                the whole process...”</p>
                            <span class="author">Sarah Johnson</span>
                            <span class="work">Web Design Media</span>
                        </div>
                    </div>
                    <div class="client clearfix">
                        <img class="photo-client" src="img/photo2.jpg">
                        <div class="quote-wrapper">
                            <p class="quote">“I only have good things to say about Design Studios - I highly recommend
                                them for promotional work...”</p>
                            <span class="author">Kate Saunders</span>
                            <span class="work">Software Solutions</span>
                        </div>
                    </div>
                    <div class="client clearfix">
                        <img class="photo-client" src="img/photo1.jpg">
                        <div class="quote-wrapper">
                            <p class="quote">“Design Studios have increased traffic to my site by over 200% in just
                                a few months - greate work guys!”</p>
                            <span class="author">Dave Cole</span>
                            <span class="work">Global Web Services</span>
                        </div>
                    </div>
                </div>
                <div class="feature-wrapper">
                    <h4 class="h-add">Featured Project</h4>
                    <h5 class="h-sub-add">Yoshi Online Marketing Campaign</h5>
                    <img class="img-add" src="img/img1.png">
                    <p class="text-add"> This project involved organizing an online marketing campaign to help promote
                        the Yoshi brand. The result of this campaign resulted in a 200% increase in Yoshi’s sales
                        over a six month period of time.
                    </p>
                    <a class="link-add" href="">Click Here For More Details &gt;&gt;</a>
                </div>
            </section>
        </div>
        <footer class="footer">
            <div class="footer-content">
                <span class="text-footer">Copyright © 2010 ~ Design Studios ~ All Rights Reserved</span>
            </div>
        </footer>
    </div>

</div>
<script src="js/jquery-1.12.4.min.js"></script>
<script src="js/scripts.js"></script>
</body>
</html>