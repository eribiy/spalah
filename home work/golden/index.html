<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
    <title>Golden</title>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
    <link href="styles/reset.css" rel="stylesheet">
    <link href="styles/style.css" rel="stylesheet">
<body>
<div class="wrapper">
    <header class="header">
        <div class="container container-header clearfix">
            <div class="logo">
                <a href="#">
                    <img src="images/Logo.png">
                </a>
            </div>
            <ul class="nav">
                <li><a href="#">Home</a></li>
                <li><a href="#">Services</a></li>
                <li><a href="#">Portfolio</a></li>
                <li><a href="#">About</a></li>
                <li><a href="#">Contact</a></li>
            </ul>
            <div class="mobile-nav-wrapper">
                <a class="mobile-btn">
                    <span></span>
                </a>
                <ul class="mobile-nav">
                    <li><a href="#">Home</a></li>
                    <li><a href="#">Services</a></li>
                    <li><a href="#">Portfolio</a></li>
                    <li><a href="#">About</a></li>
                    <li><a href="#">Contact</a></li>
                </ul>
            </div>
        </div>
    </header>
    <section class="home">
        <div class="container">
            <h3 class="h3">Welcome To Our Studio!</h3>
            <h1 class="h1">Itâ€™s nice to Meet you</h1>
            <form>
                <button class="button">Tell Me More</button>
            </form>
        </div>
    </section>
    <section class="services">
        <div class="container container-main">
            <h2 class="title">Services</h2>
            <h6 class="sub-title">Proin iaculis purus consequat sem cure.</h6>
            <div class="services-row">
                <div class="services-item first">
                    <h4 class="column-title">E-Commerce</h4>
                    <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit </p>
                </div>
                <div class="services-item second">
                    <h4 class="column-title">Responsive Web</h4>
                    <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit </p>
                </div>
                <div class="services-item third">
                    <h4 class="column-title">Web Security</h4>
                    <p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit </p>
                </div>
            </div>
        </div>
    </section>
    <section class="portfolio">
        <div class="container container-main">
            <h2 class="title">Our portfolio</h2>
            <h6 class="sub-title">Proin iaculis purus consequat sem cure.</h6>
            <div class="portfolio-row">
                <div class="portfolio-item">
                    <img src="images/ebony.jpg">
                    <h4 class="column-title">Ebony & Ivory</h4>
                    <span>Branding</span>
                </div>
                <div class="portfolio-item">
                    <img src="images/stationary.jpg">
                    <h4 class="column-title">Smart Stationary</h4>
                    <span>Print Design</span>
                </div>
                <div class="portfolio-item">
                    <img src="images/poster.jpg">
                    <h4 class="column-title">Clever Poster</h4>
                    <span>Print Design</span>
                </div>
            </div>
            <div class="portfolio-row">
                <div class="portfolio-item">
                    <img src="images/record.jpg">
                    <h4 class="column-title">Vinyl Record</h4>
                    <span>Product Mock-Up</span>
                </div>
                <div class="portfolio-item">
                    <img src="images/tamplate.jpg">
                    <h4 class="column-title">Treehouse Template</h4>
                    <span>Web Design</span>
                </div>
                <div class="portfolio-item">
                    <img src="images/burned-logo.jpg">
                    <h4 class="column-title">Burned Logo</h4>
                    <span>Branding</span>
                </div>
            </div>
        </div>
    </section>
    <section class="about">
        <div class="container container-main">
            <h2 class="title">About Us</h2>
            <h6 class="sub-title">Proin iaculis purus consequat sem cure.</h6>
            <div class="about-row clearfix">
                <div class="about-item">
                    <h4 class="column-title column-title-left">July 2010<br>Our Humble Beginnings</h4>
                    <p class="content content-left">Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt. Proin
                        iaculis purus consequat sem cure digni
                        ssim. Donec porttitora entum suscipit.</p>
                </div>
                <div class="about-item-center">
                    <img class="img-round" src="images/img1.png">
                    <img class="img-line" src="images/vertical-line.png">
                </div>
                <div class="about-item about-item-free">
                </div>
            </div>
            <div class="about-row clearfix">
                <div class="about-item about-item-free">
                </div>
                <div class="about-item-center">
                    <img class="img-round" src="images/image2.png">
                    <img class="img-line" src="images/vertical-line.png">
                </div>
                <div class="about-item">
                    <h4 class="column-title column-title-right">January 2011<br>Facing Startup Battles</h4>
                    <p class="content content-right">Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt. Proin
                        iaculis purus consequat sem cure digni
                        ssim. Donec porttitora entum suscipit aenean
                        rhoncus posuere odio in tincidunt.</p>
                </div>
            </div>
            <div class="about-row clearfix">
                <div class="about-item">
                    <h4 class="column-title column-title-left">December 2012<br>Enter The Dark Days</h4>
                    <p class="content content-left">Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt. Proin
                        iaculis purus consequat sem cure digni.</p>
                </div>
                <div class="about-item-center">
                    <img class="img-round" src="images/image3.png">
                    <img class="img-line" src="images/vertical-line.png">
                </div>
                <div class="about-item about-item-free">
                </div>
            </div>
            <div class="about-row clearfix">
                <div class="about-item about-item-free">
                </div>
                <div class="about-item-center">
                    <img class="img-round" src="images/image4.png">
                    <img class="img-line" src="images/vertical-line.png">
                </div>
                <div class="about-item">
                    <h4 class="column-title column-title-right">February 2014<br>Our Triumph</h4>
                    <p class="content content-right">Proin iaculis purus consequat sem cure
                        digni ssim. Donec porttitora entum suscipit
                        aenean rhoncus posuere odio in tincidunt. Proin
                        iaculis purus consequat sem cure digni
                        ssim. Donec porttitora entum suscipit aenean.</p>
                </div>
            </div>
            <div class="about-row clearfix">
                <div class="about-item about-item-free">
                </div>
                <div class="about-item-center">
                    <a href="#"><img class="img-round" src="images/container.png"></a>
                </div>
                <div class="about-item">
                </div>
            </div>
        </div>
    </section>
    <section class="team">
        <div class="container container-main">
            <h2 class="title">our amazing team</h2>
            <h6 class="sub-title">Proin iaculis purus consequat sem cure.</h6>
            <div class="team-row">
                <div class="team-item">
                    <img src="images/team.png">
                    <h4 class="column-title">Kimberly Thompson</h4>
                    <span>Marketer</span>
                    <div class="icon-wrap">
                        <a class="icon-link" href="#"><i class="fa fa-twitter icon icon-twitter" aria-hidden="true"></i></a>
                        <a class="icon-link" href="#"><i class="fa fa-facebook icon icon-facebook"
                                                         aria-hidden="true"></i></a>
                        <a class="icon-link" href="#"><i class="fa fa-pinterest-p icon icon-pinterest"
                                                         aria-hidden="true"></i></a>
                    </div>
                </div>
                <div class="team-item">
                    <img src="images/team.png">
                    <h4 class="column-title">Rico Massimo</h4>
                    <span>Coder</span>
                    <div class="icon-wrap">
                        <a class="icon-link" href="#"><i class="fa fa-twitter icon icon-twitter" aria-hidden="true"></i></a>
                        <a class="icon-link" href="#"><i class="fa fa-facebook icon icon-facebook"
                                                         aria-hidden="true"></i></a>
                        <a class="icon-link" href="#"><i class="fa fa-google-plus icon icon-google-plus"
                                                         aria-hidden="true"></i></a>
                    </div>
                </div>
                <div class="team-item">
                    <img src="images/team.png">
                    <h4 class="column-title">Uku Mason</h4>
                    <span>Graphic Designer</span>
                    <div class="icon-wrap">
                        <a class="icon-link" href="#"><i class="fa fa-facebook icon icon-facebook"
                                                         aria-hidden="true"></i></a>
                        <a class="icon-link" href="#"><i class="fa fa-pinterest-p icon icon-pinterest"
                                                         aria-hidden="true"></i></a>
                    </div>
                </div>
            </div>
            <h6 class="sub-title team-description">Proin iaculis purus consequat sem cure digni ssim donec porttitora
                entum suscipit
                aenean rhoncus posuere odio in tincidunt proin iaculis.</h6>
        </div>
    </section>
    <section class="logos">
        <div class="container container-logos">
            <div class="logos-row">
                <img class="img-logo" src="images/envato.png">
                <img class="img-logo" src="images/wordpress.png">
                <img class="img-logo" src="images/tuts+.png">
                <img class="img-logo" src="images/microlancer-copy.png">
            </div>
        </div>
    </section>
    <section class="contact">
        <div class="container container-main">
            <h2 class="title">Contact us</h2>
            <h6 class="sub-title">Proin iaculis purus consequat sem cure.</h6>
            <form>
                <div class="contact-raw">
                    <div class="contact-column">
                        <input class="contact-inp" type="text" placeholder="your Name *">
                        <input class="contact-inp" type="email" placeholder="your e-mail *">
                        <input class="contact-inp" type="text" placeholder="subject">
                    </div>
                    <div class="contact-column">
                        <textarea class="contact-massage" placeholder="Your message *"></textarea>
                    </div>
                </div>
                <button class="button">Send Message</button>
            </form>
        </div>
    </section>
    <footer class="footer">
        <div class="container container-footer">
            <div class="footer-raw">
                <div class="footer-column">
                    <span>Â© Copyright 2014 FreebiesXpress.com</span>
                </div>
                <div class="footer-column clearfix">
                    <a class="icon-link" href="#"><i class="fa fa-twitter icon icon-twitter" aria-hidden="true"></i></a>
                    <a class="icon-link" href="#"><i class="fa fa-facebook icon icon-facebook"
                                                     aria-hidden="true"></i></a>
                    <a class="icon-link" href="#"><i class="fa fa-pinterest-p icon icon-pinterest"
                                                     aria-hidden="true"></i></a>
                    <a class="icon-link" href="#"><i class="fa fa-google-plus icon icon-google-plus"
                                                     aria-hidden="true"></i></a>
                </div>
            </div>
        </div>
    </footer>
</div>
<script src="js/jquery-1.12.4.min.js"></script>
<script src="js/scripts.js"></script>
</body>
</html>